docs/Operators.md

```diff
@@ -15137,20 +15137,67 @@ This version of the operator has been available since version 1 of the default O
 #### Examples
 
 <details>
-<summary>unsqueeze</summary>
+<summary>unsqueeze_one_axis</summary>
 
 ```python
+x = np.random.randn(3, 4, 5).astype(np.float32)
+
+for i in range(len(x) + 1):
```

Comment:
```suggestion
for i in range(x.ndim):
```
maybe?
```
print(np.expand_dims(x, axis=len(x) + 1).shape)
```
produces
```
(3,4,5,1)
```
which is an undefined behavior in ONNX Unsqueeze.

Reply:
I'll make this `x.ndim + 1` if that is okay? Based on your other comment it seems like you are happy that this is defined behaviour.